public/typescript/functions.ts

```diff
@@ -100,9 +100,12 @@ export const isWebAuthnSupported = () => typeof navigator.credentials !== 'undef
 /**
  * Simple hashing function for message to error codes.
  */
-export const toSimpleHash = (input: string): number => reduce(
-    // tslint:disable-next-line:no-bitwise
-    (hash, char) => Math.abs(((hash << 5) - hash) + char.charCodeAt(0)),
-    0,
-    splitEvery(1, replace(/".*?"|'.*?'/g, '', input)),
-  );
+export const createArtCode = (input: string): string => `F${reduce(
```

Comment:
Maybe using the term `Art` is not fully warranted here. Your error string is different from an actual Art error code one some levels.

1. You do not utilize the crc32 calculation (I don't say you should)
2. Your error code string is longer
3. It's prepended with the F character

Consider naming it simply. 'createErrorCode'